charts/akeyless-api-gateway/values.yaml

```diff
@@ -149,6 +149,33 @@ akeylessUserAuth:
 #  - customer-fragments
 existingSecret: 
 
+
+# Specifies an initial configuration for the defaults section
+defaultsConf:
+  # defaultSamlAccessId:
+  # defaultOidcAccessId:
+  # defaultEncryptionKey:
+  # defaultSecretLocation:
+
+
+# Specifies an initial configuration for the caching section
+cachingConf:
+  # cacheEnable: <true/false>
+  # cacheTTL:
```

Comment:
Add description to cacheTTL like: ttl in minutes
also proActiveCacheMinimumFetchingTime
and proActiveCacheDumpInterval

Reply:
Done